tests/fixtures/squares.sql

```diff
@@ -26,6 +27,8 @@ BEGIN
 
     -- We want bbox divided up into depth*depth squares per bbox,
     -- so what is the width of a square?
+    depth := coalesce((query_params ->> 'depth')::int, 2);
```

Comment:
`example`

`depth` might not be present in the query_params, but we need a default